.github/workflows/build-deploy.yml

```diff
@@ -98,12 +98,15 @@ jobs:
           if (context.eventName === "pull_request") {
             return "pr-" + context.issue.number;
           }
+          if (context.eventName === "release") {
```

Comment:
Do we release Hotfixes for old versions? Or are we always increasing the version number? In the hotfix situation of a previous major version we would have problems with setting latest just by checking for a release type.

Reply:
I think that just a increased number will be used then. 
@krusche What's your thoughts on this?

Reply:
According to the Security Policy, we do not provide fixes for old versions (and I also have never seen such fix).
https://github.com/ls1intum/Artemis/blob/develop/SECURITY.md